lb3/task2/lb3task2.c

```diff
@@ -0,0 +1,246 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <stdarg.h>
+
+#define EPSILON 1e-10
+
+typedef enum {
+    SUCCESS = 0,
+    NULL_POINTER_ERROR,
+    DIMENSION_ERROR,
+    MEMORY_ALLOCATION_ERROR,
+    MATRIX_ERROR
+} ErrorCode;
+
+typedef struct {
+    int dimension;
+    double* coordinates;
+} Vector;
```

Comment:
В Си обычно структуры не называют с большой буквы

```suggestion
} vector_t;
```